src/Symfony/Bundle/FrameworkBundle/Command/AboutCommand.php

```diff
@@ -28,6 +29,18 @@ class AboutCommand extends ContainerAwareCommand
 {
     /**
      * {@inheritdoc}
+     *
+     * @deprecated since version 3.4, to be removed in 4.0
+     */
+    protected function getContainer()
```

Comment:
yes it does: it informs the readers about which major version may take advantage of the final. "final in 3.*" means "not really final until 4"